SimpleBrowser/Browser.cs

```diff
@@ -461,7 +473,9 @@ public static void ClearWindows()
         public void Close()
         {
             navigationHistory = null;
-            _allWindows.Remove(this);
+
+            if (_allWindows.Contains(this))

```

Comment:
In what cases would _allWindows not contain _this_?

Reply:
If you open child windows from frames it will call CreateChildBrowser which passes the _allWindows. Then when closing the parent it will close all the frames (Dispose-method). In that case it might already have been removed (depending of the call order).